nodejs/graphql/handlers/graphql.handler.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config({path: '.env'});
+
+const Curl = require('node-libcurl').Curl;
+const urlHandler = require('../../handlers/url.handler');
+
+class GraphQlHandler {
+    static async fetchData() {
+        const curl = new Curl();
+        const url = urlHandler.getGraphQLMainnetUrl();
+        const apiKey = process.env.ARKHIA_API_KEY;
+        const postData = JSON.stringify({
+            query: '{ account(input: { entityId: { shard: 0, realm: 0, num: 98 } }) { alias autoRenewAccount { entityId { shard realm num } } autoRenewPeriod balance createdTimestamp declineReward deleted entityId { shard realm num } expirationTimestamp id key maxAutomaticTokenAssociations memo nonce obtainer { entityId { shard realm num } } pendingReward receiverSigRequired stakedAccount { entityId { shard realm num } } stakePeriodStart timestamp { from to } type } }',
+            variables: {},
+        });
+
+        return new Promise((resolve, reject) => {
```

Comment:
This gets the job done, but curious why you chose to use a Promise instead of the async await?

Reply:
Same question for me

Reply:
@aaron-bcw @daniel-bcw 

I used `Promise` because it worked better with this Curl request; I initially tried `Aysnc` and `await` was getting errors with this particular library(node-lib); maybe it was the way I was implementing it but this worked smoothly for me